pyfolio/txn.py

```diff
@@ -146,6 +147,8 @@ def adjust_returns_for_slippage(returns, positions, transactions,
     return adjusted_returns
 
 
+@deprecated(msg='pyfolio.get_turnover is deprecated! '
```

Comment:
Do we need to move turnover? I don't think we're using this downstream anymore.

Reply:
we use it in perf attrib to determine whether to warn the user, plus I figured it counts as the sort of low level calculation that would make sense in empyrical

Reply:
Ah, I see, this is a dependency of `perf_attrib`.  I commented in https://github.com/quantopian/empyrical/pull/76#discussion_r165421747 that I'm not sure we want all the validation/warning logic in zipline (where we don't ever expect to have missing or misaligned data, and if we do at least on Quantopian a user won't see the warnings emitted by perf attrib b/c they'll happen at the end of a simulation).  If we just move the performance attribution calculations to empyrical, then I don't think we need to move this, since we're currently only using turnover as a validation step.

Reply:
The only reason I'm a little hesitant to move this is that the format we use for transactions is currently used only by pyfolio, so right now we could change it and not break much.  Moving this into empyrical means that the format now crosses a project boundary, which makes it harder to change in the future.